frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/PipelineRunDrawerRightTabs.tsx

```diff
@@ -22,34 +20,21 @@ enum PipelineRunNodeTabs {
 const PipelineRunNodeTabsTitles = {
   [PipelineRunNodeTabs.INPUT_OUTPUT]: 'Input / Output',
   [PipelineRunNodeTabs.DETAILS]: 'Details',
-  [PipelineRunNodeTabs.VOLUMES]: 'Volumes',
   [PipelineRunNodeTabs.LOGS]: 'Logs',
 };
 
 type PipelineRunDrawerRightTabsProps = {
-  task: PipelineRunTaskDetails;
-  taskReferences: TaskReferenceMap;
-  parameters?: PipelineRunTaskParam[];
+  task: PipelineTask;
 };
 
-const PipelineRunDrawerRightTabs: React.FC<PipelineRunDrawerRightTabsProps> = ({
-  task,
-  taskReferences,
-  parameters,
-}) => {
+const PipelineRunDrawerRightTabs: React.FC<PipelineRunDrawerRightTabsProps> = ({ task }) => {
   const [selection, setSelection] = React.useState(PipelineRunNodeTabs.INPUT_OUTPUT);
 
   const tabContents: Record<PipelineRunNodeTabs, React.ReactNode> = {
-    [PipelineRunNodeTabs.INPUT_OUTPUT]: (
-      <SelectedNodeInputOutputTab
-        taskReferences={taskReferences}
-        task={task}
-        parameters={parameters}
-      />
-    ),
+    [PipelineRunNodeTabs.INPUT_OUTPUT]: <SelectedNodeInputOutputTab task={task} />,
     // [PipelineRunNodeTabs.VISUALIZATIONS]: <>TBD 2</>,
     [PipelineRunNodeTabs.DETAILS]: <SelectedNodeDetailsTab task={task} />,
-    [PipelineRunNodeTabs.VOLUMES]: <SelectedNodeVolumeMountsTab task={task} />,
+    // [PipelineRunNodeTabs.VOLUMES]: <SelectedNodeVolumeMountsTab task={task} />,
```

Comment:
why is volumes removed

Reply:
I have not found a way to show them, if you have a Pipeline for them, let me know!

Reply:
Found one, will implement again

Reply:
https://github.com/opendatahub-io/odh-dashboard/pull/2598#issuecomment-2000047662 yes, I am aware, fixing. They do weird spec things when it has volumes.